### ISSUE TITLE
Links on relationship properties / inherited relationships on sync instances may not point to "true" entities in the database

### ISSUE THREAD

**Issue by RafaPolit** [huridocs/uwazidocs#4527]
Facts:
- Our "relationship" metadata properties render a label with a link to navigating to (or showing in the side panel) that entity's data
- We allow partial syncing of instances 

Therefore:
- we could end up with a collection where we show an entity that has values for a particular relationship property that points to an entity which DOES NOT exist on the collection

On such cases, we need to decide what to do with the label / url?

Options that come to mind:
- show the label but not make it into a URL, as you have no extra info to show
- keep the link and tell the user that the extra info is not present, which doesn't seem ideal to me
- hide the value, since the target entity is not in the collection.  This breaks our rule that "entity is the source of truth of permissions / visible values".  I wouldn't do this, as the admin has the option to NOT sync that property if they want this behavior to happen.

### Observations

Option would require either that:
- the backend sends the URL along with the label taking into account existence or not of the related entity (probably the preferred method), or...
- to have the client perform API calls in order to determine if entities exist in order to show or not the URL

**Comment by txau** [comment]
The best option seems to be `show the label but not make it into a URL, as you have no extra info to show` but it is unclear how to achieve this with an elegant option. The proposed solution `the backend sends the URL along with the label` seems good enough and has another positive upside: it can also be used in the case of a not-synced Uwazi when the user has no permissions to see the target entity.

That said, resolving these "entity exists/I have permission" queries can add a lot of overhead when the links are shown in cards and tables in library view, so probably we should deactivate these links (currently active) in library view and side panel.

Currently such links to unauthorized entities toss a 404, so we are talking about almost the same scenario as sync with missing entities.

**Comment by RafaPolit** [comment]
Exactly, backend-decision of which data to send seems like the more robust scenario.  True about the permissions, so this is broader than sync.
